docs/rfcs/009-network-architecture/README.md

```diff
@@ -0,0 +1,44 @@
+# RFC 009: Network Architechture
```

Comment:
minor: I’m already using 009 in https://github.com/wellcometrust/platform/pull/3138